homeassistant/components/openweathermap/config_flow.py

```diff
@@ -107,6 +111,12 @@ async def async_step_init(self, user_input=None):
     def _get_options_schema(self):
         return vol.Schema(
             {
+                vol.Optional(
+                    CONF_LATITUDE, default=self.hass.config.latitude
+                ): cv.latitude,
+                vol.Optional(
+                    CONF_LONGITUDE, default=self.hass.config.longitude
+                ): cv.longitude,
```

Comment:
The unique ID of this configuration flow is created by these. Making them optional, renders the unique ID invalid (which needs to be addressed).